### PR TITLE
Feature/chinedu/user profile

### PR DESCRIPTION
**Pull request by ChineduOguejiofor**
#### What does this PR achieve?
- Started User Profile
- Added the forms

#### Screenshots:
userProfile

### CODE REVIEW

File: src/components/userProfile/components/FormItem.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    });
+    const onChange = name => e => {
+        setDetails({ ...details, [name]: e.target.value });
+    };
+    const handleSubmit = e => {
+        e.preventDefault();
+    };
+    return (
+        <FormComp handleSubmit={handleSubmit} onChange={onChange} />
+    );
```
Comment (Way2nnadi) [inline src/components/userProfile/components/FormItem.jsx]: you only need the parentheses if you component is defined on multiple lines. For this just return the component on one line
```
return  <FormComp handleSubmit={handleSubmit} onChange={onChange} />;
```

File: src/components/userProfile/components/UserProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <div>
+        <Card className="border-radius">
+            <Title level={4}>
+                {PROFILE}
+            </Title>
+            <Divider />
+
+            <DisplayRow />
+        </Card>
+    </div>
```
Comment (Way2nnadi) [inline src/components/userProfile/components/UserProfile.jsx]: Use a fragment here

Comment (ChineduOguejiofor) [reply src/components/userProfile/components/UserProfile.jsx]: Updated 
Thanks

File: src/components/userProfile/components/FormItem.jsx (hunk 1 above)
Comment (ChineduOguejiofor) [reply src/components/userProfile/components/FormItem.jsx]: parentheses removed
thanks

**Pull request merged**